applications/frontend/views.py

```
@@ -0,0 +1,55 @@
+        return context
+
+
+class GitHubCreateIssueEndPoint(FormView):
+    form_class = InsultReviewForm
+
+    def post(self, request: HttpRequest, *args, **kwargs) -> Response | None:
+        form = InsultReviewForm(request.POST)
+        if form.is_valid():
+            try:
+                return self.format_issue(form)
+            except Exception as e:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)